Remotion/Validation/Core/Attributes/Validation/LengthValidationAttribute.cs

```diff
@@ -52,7 +53,11 @@ protected override IEnumerable<IPropertyValidator> GetValidators (IPropertyInfor
       ArgumentUtility.CheckNotNull("validationMessageFactory", validationMessageFactory);
 
       IPropertyValidator validator;
-      if (string.IsNullOrEmpty(ErrorMessage))
+      if (MinLength == 0 && MaxLength == 0)
+      {
+        return Enumerable.Empty<IPropertyValidator>();
+      }
+      else if (string.IsNullOrEmpty(ErrorMessage))
       {
         validator = PropertyValidatorFactory.Create(
             property,
```

Comment:
okay, there is no good way to do this inside `CreateValidator` apprently, atleast in the existing code structure. Because of the way PropertyValidatorFactory works, there is also no way to change this without introducing a NullValidator object that is then filtered. 

Basically, this would look like this:
CreateValidator returns a NullValidator (i.e. a null object) for Min==Max==0 and in GetValidators we check if the result is the NullValidator and return Enumerable.empty or Enumerable.Singleton based on this result check.

Advantage of that solution:
We only have one place where we decide how to handle Min/Max Length. Is it worth the back and forth? Perhaps. I think we should take a moment and do a separate version of the code with the alternative implementation. that way, we can see how it feels like.

Should we ever decide to also gracefully handle other incorrect configurations, this might actually be necessary.


Code style for the current solution:
please make the IF as an exit condition because it has nothing to do with the original IF, but rather cuts out before things get interesting.

Reply:
I guess for now we want to revert it to my first apporach d92963a0fae07f887e622052d8e6e7429f831db3. Do we want to implement the other version of the code in this pr or should I create a ticket for this?

Reply:
> I guess for now we want to revert it to my first apporach https://github.com/re-motion/Framework/commit/d92963a0fae07f887e622052d8e6e7429f831db3.
Yep, sounds good. checked the old code and yeah, I though it was part of the helper method not the "outer" method.

> Do we want to implement the other version of the code in this pr or should I create a ticket for this?
just do a quick demo in the current PR based on a copy of the current file. it's just so we can check out the code/discuss and then decide which to keep.